Commands/InteractionServiceHandler.cs

```diff
@@ -52,11 +53,11 @@ public static async Task RegisterModulesAsync()
 
 			await _interactionService.RegisterCommandsGloballyAsync();
 			_client!.InteractionCreated += OnInteractionCreatedAsync;
-			await Program.LogAsync("InteractionServiceManager", "Modules registered successfully", LogSeverity.Info);
+			Log.Information("Modules registered successfully, InteractionServiceManager");
 		}
 		catch (Exception e)
 		{
-			await Program.LogAsync("InteractionServiceManager", $"Error registering modules. ({e})", LogSeverity.Critical);
+			Log.Fatal($"Error registering modules. ({e})");
```

Comment:
Not sure if the user needs the whole error.
Could be dependent on the debug bool from the config for example

```suggestion
			Log.Fatal($"Error registering modules: {e.Message}");
```
Or dependent on the debug bool from the config, though the current implementation would require quite a bit of code
```suggestion
						Log.Fatal($"Error registering modules: {(Program.config != null && Program.config.debug ? e : e.Message)}");
```
Or the additional information could be provided with a separate debug log, requiring less code, but the message would be split into 2.
All of this could be implemented into the other error logs

Reply:
The second option is of course voluminous in terms of code, but extremely correct in terms of its functionality, so it is necessary to choose it.